<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pageLogin">
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>172</width>
        <height>268</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>170</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>170</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="Imagens.qrc">:/Icons/Imgs/devil.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditNameUser">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>25</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Nome de usu√°rio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPassword">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>25</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Senha</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLogin">
         <property name="minimumSize">
          <size>
           <width>170</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pageProgram">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>180</width>
        <height>490</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>330</width>
        <height>450</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>450</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>330</width>
        <height>450</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>328</width>
         <height>448</height>
        </rect>
       </property>
       <widget class="QLabel" name="labelMessages">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>310</width>
          <height>431</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>310</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>310</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>331</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditSend">
         <property name="placeholderText">
          <string>Digite sua mensagem</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSend">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Imagens.qrc"/>
 </resources>
 <connections/>
</ui>
